integration/networktest/env/dev_network.go

```diff
@@ -0,0 +1,34 @@
+package env
+
+import (
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+	"github.com/obscuronet/go-obscuro/integration/simulation/devnetwork"
+)
+
+type devNetworkEnv struct{}
+
+func (d *devNetworkEnv) Prepare() (networktest.NetworkConnector, func(), error) {
+	devNet := devnetwork.DefaultDevNetwork()
+	devNet.Start()
```

Comment:
Likely that there is a swallowed error here

Reply:
Not swallowed, but panicked. The dev network stuff panics if it fails during start-up  at the moment, since it's test code but maybe that should be changed.

Reply:
Yep, worth surfacing all errors.